Problem compiling ImageMagick6-7 from source

Under previous and current versions of ImageMagic 6.x and 7.x I'm getting errors in compiling from source, using an Asus Flip C101PA (new). Since this previously worked on a different Chromebook, it may be that the lmza library or some other dependency has broken rather than ImageMagick itself.

Here is the error message:
~~~
MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `lzma_easy_encoder'
MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `lzma_code'
MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `lzma_auto_decoder'
MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `lzma_end'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:7057: utilities/magick] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/ImageMagick-7.0.7-22.tar.xz.dir/ImageMagick-7.0.7-22'
make: *** [Makefile:5185: all] Error 2
imagemagick7 failed to install: `make V=0 -j6` exited with 2
~~~

I've previously submitted changes to three files to update to the current versions available from ImageMagick, however I've also tested with earlier versions found on archived ftp locations, with the same error message.

## Versions and system information

~~~
chromebrew: 0.4.3

CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4613139
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={564AC308-CBD5-485D-9EF5-EB97DBB5F264}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={564AC308-CBD5-485D-9EF5-EB97DBB5F264}
CHROMEOS_RELEASE_BOARD=bob-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=76
CHROMEOS_RELEASE_BUILDER_PATH=bob-release/R64-10176.76.0
CHROMEOS_RELEASE_BUILD_NUMBER=10176
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=64
CHROMEOS_RELEASE_DESCRIPTION=10176.76.0 (Official Build) stable-channel bob
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=10176.76.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10176.76.0
~~~

--- a/packages/imagemagick.rb
+++ b/packages/imagemagick.rb
@@ -20,16 +20,32 @@ class Imagemagick < Package
         source_url 'https://www.imagemagick.org/download/releases/ImageMagick-6.9.9-39.tar.xz'
         source_sha256 '56130c16790d8d38c3a1888cc07074d5b50ffb718243e0c56e4c88839edbf931'
         binary_url ({
+          aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-armv7l.tar.xz',
+           armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-armv7l.tar.xz',
+             i686: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-i686.tar.xz',
+           x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-x86_64.tar.xz',
         })
         binary_sha256 ({
+          aarch64: '071bea4fdc809f539bf50cac533116d2e503930950cec5a38d5327bc0d079cfd',
+           armv7l: '071bea4fdc809f539bf50cac533116d2e503930950cec5a38d5327bc0d079cfd',
+             i686: 'ece2e339acfb29ae048e7042a68f673b442ef9f024490461906f2e205676ecdf',
+           x86_64: '624ac10730c4b99c23a3049e0ec2a0a9ebdbd4662e7a3e2be8829636e3f187c4',
         })
         break
       when '7'
         source_url 'https://www.imagemagick.org/download/releases/ImageMagick-7.0.7-27.tar.xz'
         source_sha256 '543776f09d69e3ca29b1b83a9c0223185ba26bec673593840b4d7face6ea253b'
         binary_url ({
+          aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-armv7l.tar.xz',
+           armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-armv7l.tar.xz',
+             i686: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-i686.tar.xz',
+           x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-x86_64.tar.xz',
         })
         binary_sha256 ({
+          aarch64: '2b60e59633eeb052949a7792bd8fde360db7a6c34e04c7ce6f7c1fadfe5411ea',
+           armv7l: '2b60e59633eeb052949a7792bd8fde360db7a6c34e04c7ce6f7c1fadfe5411ea',
+             i686: '8c549ac27f1d0a6248411662dc64d9cb96df5d00ef1808ec802f1dac875c0a2d',
+           x86_64: 'd37404daaf575bf898be65c65aed1761d32031abaffd9eac2d2a6c29696febda',
         })
         break
       when '0'

--- a/packages/imagemagick6.rb
+++ b/packages/imagemagick6.rb
@@ -3,13 +3,21 @@ require 'package'
 class Imagemagick6 < Package
   description 'Use ImageMagick to create, edit, compose, or convert bitmap images.'
   homepage 'http://www.imagemagick.org/script/index.php'
-  version '6.9.9-36'
-  source_url 'https://www.imagemagick.org/download/releases/ImageMagick-6.9.9-36.tar.xz'
-  source_sha256 'a86dc03ef8981e2dec83657fce3b4976347a537d785cc8e08af85af22a1b1bda'
+  version '6.9.9-39'
+  source_url 'https://www.imagemagick.org/download/releases/ImageMagick-6.9.9-39.tar.xz'
+  source_sha256 '56130c16790d8d38c3a1888cc07074d5b50ffb718243e0c56e4c88839edbf931'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick6-6.9.9-39-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '071bea4fdc809f539bf50cac533116d2e503930950cec5a38d5327bc0d079cfd',
+     armv7l: '071bea4fdc809f539bf50cac533116d2e503930950cec5a38d5327bc0d079cfd',
+       i686: 'ece2e339acfb29ae048e7042a68f673b442ef9f024490461906f2e205676ecdf',
+     x86_64: '624ac10730c4b99c23a3049e0ec2a0a9ebdbd4662e7a3e2be8829636e3f187c4',
   })
 
   depends_on 'freetype'

--- a/packages/imagemagick7.rb
+++ b/packages/imagemagick7.rb
@@ -8,8 +8,16 @@ class Imagemagick7 < Package
   source_sha256 '543776f09d69e3ca29b1b83a9c0223185ba26bec673593840b4d7face6ea253b'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/imagemagick7-7.0.7-27-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '2b60e59633eeb052949a7792bd8fde360db7a6c34e04c7ce6f7c1fadfe5411ea',
+     armv7l: '2b60e59633eeb052949a7792bd8fde360db7a6c34e04c7ce6f7c1fadfe5411ea',
+       i686: '8c549ac27f1d0a6248411662dc64d9cb96df5d00ef1808ec802f1dac875c0a2d',
+     x86_64: 'd37404daaf575bf898be65c65aed1761d32031abaffd9eac2d2a6c29696febda',
   })
 
   depends_on 'freetype'

--- a/packages/lzma.rb
+++ b/packages/lzma.rb
@@ -3,21 +3,21 @@ require 'package'
 class Lzma < Package
   description 'LZMA Utils are legacy data compression software with high compression ratio.'
   homepage 'https://tukaani.org/lzma/'
-  version '4.32.7'
+  version '4.32.7-1'
   source_url 'https://tukaani.org/lzma/lzma-4.32.7.tar.bz2'
   source_sha256 '618e54513993b3a153fa1c150fccdf25788c72b36e84ab4db71911083531cf6a'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/lzma-4.32.7-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '6de4cd3976b06fe144d92a2abeb2f8924df65a353b6cf0d76f5da0aa97188c32',
-     armv7l: '6de4cd3976b06fe144d92a2abeb2f8924df65a353b6cf0d76f5da0aa97188c32',
-       i686: 'a7262e79b86793ccab69c8cb787fad3b5193dc6715134904adf1b40ceda6f7ab',
-     x86_64: '807699df57e6feda914ab0aaef80c15e4fc57e7ad75c0c76e79d5ef7eaabfc26',
+    aarch64: 'cb62ba6e36b24cf50d0490e3ed9c35c35a76ca394f98e6c572bf3e49ab48f205',
+     armv7l: 'cb62ba6e36b24cf50d0490e3ed9c35c35a76ca394f98e6c572bf3e49ab48f205',
+       i686: '73b723ede8d112ff824f7ab62bb5108f711d0a0cad6daf9472d2303b634e1516',
+     x86_64: 'c8a08a86689baf36796baa5c86c1507b10b0b5cdaca733737be6c59782b07b1a',
   })
 
   def self.build
@@ -27,6 +27,5 @@ class Lzma < Package
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    system "ln -s #{CREW_LIB_PREFIX}/liblzmadec.la #{CREW_DEST_LIB_PREFIX}/liblzma.la"
   end
 end

--- a/packages/radare2.rb
+++ b/packages/radare2.rb
@@ -8,8 +8,16 @@ class Radare2 < Package
   source_sha256 'e2edef4d70c7bbbb47d04002ce9d384eb2fc9c0cd4cbfde77cda8c10cae9ff24'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/radare2-2.4.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/radare2-2.4.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/radare2-2.4.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/radare2-2.4.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '7f1625aa3bbeea13de76bf122eab70130df32da119585cb0e46cea523f3e07ab',
+     armv7l: '7f1625aa3bbeea13de76bf122eab70130df32da119585cb0e46cea523f3e07ab',
+       i686: '81f78367ade45bc48632a1cfe4ebaebd9d6c549abfe3ff26a7d32116b79329ee',
+     x86_64: 'fcde3ae52954bf875ecbc650d0e0392ef80175f3df4c7b7f560aeb0535ed1969',
   })
 
   depends_on 'openssl'

--- a/packages/regex_opt.rb
+++ b/packages/regex_opt.rb
@@ -8,8 +8,16 @@ class Regex_opt < Package
   source_sha256 '128c8ba9570b1fd8a6a660233de2f5a4022740bc5ee300300709c3894413883f'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/regex_opt-1.2.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/regex_opt-1.2.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/regex_opt-1.2.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/regex_opt-1.2.4-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '6a204f3e5a2dba59affe403934fd0118f7a03fba1c9567d98688cf5723a0b3dd',
+     armv7l: '6a204f3e5a2dba59affe403934fd0118f7a03fba1c9567d98688cf5723a0b3dd',
+       i686: '367a7f0d1863da8b01cfc0fc5885c5603da46dabaf913b82a77082b02bcedd68',
+     x86_64: 'a49aaa0db5429edeb512833c3c73e96646903e8c5b2aaa0a15ef262ed62e3095',
   })
 
   def self.build

--- a/packages/selenium_server_standalone.rb
+++ b/packages/selenium_server_standalone.rb
@@ -8,8 +8,16 @@ class Selenium_server_standalone < Package
   source_sha256 '662789e2508babe0be2d5b3b84630834b7727ed1960b66567be1c73eed08433b'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/selenium_server_standalone-3.11.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/selenium_server_standalone-3.11.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/selenium_server_standalone-3.11.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/selenium_server_standalone-3.11.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '38dd130c44c9cd16deefd85201331343ef675229870ddd80f1d9a2fbc7cfd432',
+     armv7l: '38dd130c44c9cd16deefd85201331343ef675229870ddd80f1d9a2fbc7cfd432',
+       i686: '9c28cecde0b64d1570e2298605e646de730171ae6777966b90fe7fc96cec9e7f',
+     x86_64: '03c5ef09c37265bd9793bda2a1b4bf62e6b2b98c67921b81276313d78ab5396d',
   })
 
   depends_on 'jdk8'

